docker-compose.test.yml

```diff
@@ -4,10 +4,8 @@ version: "3.9"
 services:
   ihatemoney:
     build: .
-    ports:
-      - "8000:8000"
   sut:
     image: alpine
-    command: wget --spider ihatemoney:8000
+    command: sh -c 'wget -qO- ihatemoney:8000/healthcheck | grep "OK"'
```

Comment:
Why this move is needed?

This add a new route for Docker only, and I feel that just fetching the home page with checking that the return code is 200 is enough.

Reply:
As I said in the PR, /healthcheck is also usefull for monitoring, kubernetes environments ect

Not fetching the whole page is the point